torch/multiprocessing/cuda_multiprocessing.md

```diff
@@ -0,0 +1,79 @@
+# CUDA IPC Refcounting implementation explained
+
+Since shared CUDA memory belongs to the producer process, we need to take special precautions to make sure that it is stays allocated for entire shared tensor life-span.
+
+It could be done manually by syncing on an event:
```

Comment:
"If the producer process was the only process with an owning reference to a tensor (as was the case prior to PyTorch 1.1), we could manually ensure that the producer process only frees the tensor after the consumer is done with it by syncing on an event."